pkgs/applications/misc/furtherance/default.nix

```
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, appstream-glib, desktop-file-utils
+, glib, libadwaita, meson, ninja, pkg-config, python3, wrapGAppsHook4, dbus
+, gtk4, sqlite }:
+stdenv.mkDerivation rec {
+  pname = "Furtherance";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "lakoliu";
+    repo = pname;
```

If the canonical repository name has an uppercase in it, though, we should probably use that.

```suggestion
    repo = "Furtherance";
```